frontend-react/e2e/spec/all/authenticated/code-mapping-tool-page.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { CodeMappingPage } from "../../../pages/authenticated/code-mapping";
+import { test as baseTest, expect } from "../../../test";
+
+export interface CodeMappingPageFixtures {
+    codeMappingPage: CodeMappingPage;
+}
+
+const test = baseTest.extend<CodeMappingPageFixtures>({
+    codeMappingPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new CodeMappingPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe("Code Mapping Tool Page", () => {
+    test("not authenticated user", async ({ codeMappingPage }) => {
+        await codeMappingPage.page.goto(CodeMappingPage.URL_CODE_MAPPING);
+        await expect(codeMappingPage.page).toHaveURL("/login");
+    });
+
+    /*
+     *
+     * Admin User
+     *
+     * */
+    test.describe("admin user", () => {
+        test.use({ storageState: "e2e/.auth/admin.json" });
+
+        test("page loads", async ({ codeMappingPage }) => {
+            await codeMappingPage.page.goto("/onboarding/code-mapping");
+            await codeMappingPage.testHeader();
+            await expect(codeMappingPage.page.locator("footer")).toBeAttached();
+        });
+
+        // if the admin does not have a sender profile set
```

Comment:
Should we have a happy path "uploads a CSV with no errors if sender profile is set"?